payments-app-extension-custom-onsite/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,27 @@
+[[extensions]]
+name = "{{ name }}"
+type = "payments_app_extension"
+handle = "{{ handle }}"
+target = "custom_onsite"
+
+buyer_label_to_locale = []
+default_buyer_label = "Custom Onsite Payments App Extension"
+merchant_label = "Custom Onsite Payments App Extension"
+start_payment_session_url = "https://api.paymentprovider.com/endpoint"
+supports_oversell_protection = false
+confirmation_callback_url = "https://api.paymentprovider.com/confirm"
+start_refund_session_url = "https://api.paymentprovider.com/refund"
+start_capture_session_url = "https://api.paymentprovider.com/capture"
+start_void_session_url = "https://api.paymentprovider.com/void"
+supported_countries = ["US"]
+supported_payment_methods = ["visa"]
+supports_3ds = false
+supports_installments = false
+supports_deferred_payments = false
+test_mode_available = true
+api_version = "2021-07"
+multiple_capture = false
+update_payment_session_url = "https://api.paymentprovider.com/update"
+ui_extension_registration_uuid = "0e63f1a6-ed7f-41b2-8e56-e6399ac48764"
+checkout_payment_method_fields = []
+modal_payment_method_fields = []
```

Comment:
I can only see one of this fields in the partners UI (UI Extension Field Definitions) 🤔 

![image](https://github.com/Shopify/extensions-templates/assets/2975360/81108459-46ef-49a1-8ea6-e4f68f8bab31)

Reply:
Both fields are included here: 
https://github.com/Shopify/shopify/blob/main/components/payment_processing/payments_partners/app/models/payments_partners/payments_app_extensions/custom_onsite_payments_app_extension.rb#L51